submissions/strumerlz/html-css-popup/index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+        <title>HTML & CSS practice: Pop-Up Menu</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Metatag Viewport is mandatory for mobile markup.  
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Could you please add it?
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

Reply:
Fixed it.